src/test/resources/projects/dependencyResolutionManagement/projmap/src/main/resources/modid.accesswidener

```diff
@@ -0,0 +1,2 @@
+accessWidener   v1   named
+extendable   class   FluidState
```

Comment:
This isnt a valid accesswidener, I would remove it as I dont see how its realted to this test.

Reply:
Oh, I see it forces a project mapped jar, still not a valid class name

Reply:
Sorry, I do not have much experience with how the accesswidener works. This test is to force a build to use a project-mapped jar. Could you change this to a valid accesswidener? Thanks.